crates/ruff/src/rules/flake8_executable/rules/shebang_missing.rs

```diff
@@ -42,6 +44,9 @@ impl Violation for ShebangMissingExecutableFile {
 /// EXE002
 #[cfg(target_family = "unix")]
 pub(crate) fn shebang_missing(filepath: &Path) -> Option<Diagnostic> {
+    if wsl::is_wsl() {
```

Comment:
Please add a sentence here as a comment explaining why this check isn't used on WSL

Reply:
Done!